src/Overlay/Modal/Modal.tsx

```diff
@@ -0,0 +1,33 @@
+import { forwardRef } from "react";
+import * as Styled from './Modal.styled'
+import { Button } from '../../Button'
+
+type ModalProps = {
+    hasCloseBtn?: boolean;
```

Comment:
remove prop. The close button should only not show if a custom footer template is shown.

Reply:
Done.